src/pages/Checklist/index.tsx

```diff
@@ -188,58 +318,176 @@ export const Checklist = () => {
             <li className="py5">
               <Text>
                 <FormattedMessage
-                  defaultMessage="For example, you need at least ~400 GB SSD to run Geth sync on
-                    {mainnet} in February 2021, assuming you move the {ancient} directory on a
-                    different drive. Have in mind that Geth {chaindata} grows with ~1 GB per day,
-                    so you should account for enough space on your SSD and HDD until you run
-                    maintenance on the node."
+                  defaultMessage="As of {date}, ~400GB is needed for the Eth1 mainnet chain data alone (growing at ~1GB/day)."
```

Comment:
FYI @wackerow @ryancreatescopy I refactored these use of dates to use the `<FormattedDate/>` component, which should automatically translate & format dates depending on the current language. If you notice any other use of dates, please follow this pattern.